scripts/runfabtests.sh

```diff
@@ -355,15 +416,16 @@ function usage {
 	errcho -e " -v\tprint output of failing"
 	errcho -e " -vv\tprint output of failing/notrun"
 	errcho -e " -vvv\tprint output of failing/notrun/passing"
-	errcho -e " -t\ttest set(s): all,quick,unit,simple,standard,short (default quick)"
+	errcho -e " -t\ttest set(s): all,quick,unit,simple,standard,short,complex (default quick)"
```

Comment:
Yes, the default test type is "quick" which is run by travis. With "quick" option it'll run ubertest with quick.json configure file (e.g udp/quick.json).